tests/tensorflow2/test_keras.py

```diff
@@ -405,15 +405,17 @@ def test_keras_fit(out_dir, tf_eager_mode, saveall):
     helper_keras_fit(
         trial_dir=out_dir,
         hook=hook,
-        eager=tf_eager_mode,
+        run_eagerly=tf_eager_mode,
```

Comment:
the param "eager" is required for L43 above.

Reply:
Yes

Reply:
so you should pass eager too?

Reply:
`eager` is set to True by default.
`run_eagerly` indicates to the `model.compile` that the model should be run eagerly